### PR TITLE
Fixes for actual usage

### PR DESCRIPTION
**Pull request by dbnicholson**
A bunch of fixes to make this actually ready to use on our server. There are still some rough edges, but this should be good enough to get things going now.

https://phabricator.endlessm.com/T17258

### CODE REVIEW

File: task.py [hunk 1]
```diff
@@ -11,6 +11,23 @@
...
+        # There's almost certainly a better way to do this. On py3
+        # there's enum, but here we'll just inline it.
+        if state == TaskState.PENDING:
+            return 'PENDING'
+        elif state == TaskState.PROCESSING:
+            return 'PROCESSING'
+        elif state == TaskState.COMPLETED:
+            return 'COMPLETED'
+        elif state == TaskState.FAILED:
+            return 'FAILED'
```
Comment (sgnn7) [inline task.py:26]: You can use a non-copy/paste enum logic here:
```
from enum import Enum
class TaskState(Enum):
      PENDING, PROCESSING, COMPLETED, FAILED = range(4)


# state = TaskState.PROCESSING

try:
    return TaskState(state).name
except:
    blah 
```

Comment (sgnn7) [reply task.py:26]: Edit: Or your logic below in `get_state_name`

**Review by sgnn7 — CHANGES_REQUESTED** [review]
@dbnicholson Just one minor comment

File: task.py (hunk 1 above)
Comment (dbnicholson) [reply task.py:26]: There's no `enum` in py2 or I would have used it :) (in the comment and commit message)

**Comment by dbnicholson** [comment]
Responded to your comment. I pushed a fix in 9ff1251 to update the `Dockerfile`, too.

**Comment by sgnn7** [comment]
CR Pass

**Pull request merged**